SimPEG/EM/Static/Utils/StaticUtils.py

```diff
@@ -937,3 +937,64 @@ def getSrc_locs(survey):
     srcMat = np.vstack(srcMat)
 
     return srcMat
+
+
+def gettopoCC(mesh, actind):
```

Comment:
This looks like a general util, not necessarily specific to static utils. Could you instead put it in "modelutils" for now? It has similar functionality to the other utils currently there.